app/commands/git/sync_doc.rb

```diff
@@ -2,10 +2,11 @@ module Git
   class SyncDoc
     include Mandate
 
-    def initialize(config, section, track: nil)
+    def initialize(config, section, git_sha, track: nil)
```

Comment:
I added the Git SHA to allow the automatically opened issues to reference the commit.